packages/zowe-explorer/src/abstract/ZoweSaveQueue.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as path from "path";
+import * as vscode from "vscode";
+import { Gui, IZoweDatasetTreeNode, IZoweTree, IZoweUSSTreeNode } from "@zowe/zowe-explorer-api";
+import * as globals from "../globals";
+import { markDocumentUnsaved } from "../utils/workspace";
+
+// Set up localization
+import * as nls from "vscode-nls";
+nls.config({
+    messageFormat: nls.MessageFormat.bundle,
+    bundleFormat: nls.BundleFormat.standalone,
+})();
+const localize: nls.LocalizeFunc = nls.loadMessageBundle();
+
+interface SaveRequest {
+    uploadRequest: (document, provider) => Promise<void>;
```

Comment:
should this be `(_document, _provider) => Promise<void>` to avoid lint warnings?
or can we update the rule to exclude `interface` definitions?